### ISSUE TITLE
dateparser does not result consistently

### ISSUE THREAD

**Issue by karthikshub** [scrapinghub/dateparser#514]
now-15m doesn't result consistent output.

>>> stmp = dateparser.parse('now-15min')
>>> stmp
datetime.datetime(2019, 4, 4, 9, 49, 23, 29148)
>>> stmp = dateparser.parse('now-15M')
>>> stmp
datetime.datetime(2019, 4, 4, 9, 49, 37, 407403)
>>> stmp = dateparser.parse('now-15m')
>>> stmp
datetime.datetime(2019, 4, 4, 9, 49, 49, 710735)
>>> stmp = dateparser.parse('now-15d')
>>> stmp
datetime.datetime(2019, 3, 20, 10, 4, 57, 439242)
>>> stmp = dateparser.parse('now-15m')
>>> stmp
datetime.datetime(2018, 1, 4, 10, 5, 3, 744759)
>>> dateparser.__version__
'0.7.1'
>>>

**Comment by Gallaecio** [comment]
I can reproduce it as well.

**Comment by rennerocha** [comment]
Same root cause as #549 . Printing [the content of this line](https://github.com/scrapinghub/dateparser/blob/master/dateparser/__init__.py#L53), we can see that the locale used changed.

```
In [3]: dateparser.parse('now-15m') 
   ...:                                                                                                                                                                     
{'date_obj': datetime.datetime(2019, 9, 3, 15, 21, 32, 569534), 'period': 'day', 'locale': 'en'}
Out[3]: datetime.datetime(2019, 9, 3, 15, 21, 32, 569534)

In [4]: dateparser.parse('now-15d') 
   ...:                                                                                                                                                                     
{'date_obj': datetime.datetime(2019, 8, 19, 15, 36, 36, 695301), 'period': 'day', 'locale': 'lb'}
Out[4]: datetime.datetime(2019, 8, 19, 15, 36, 36, 695301)

In [5]: dateparser.parse('now-15m') 
   ...:                                                                                                                                                                     
{'date_obj': datetime.datetime(2018, 6, 3, 15, 36, 40, 798523), 'period': 'month', 'locale': 'lb'}
Out[5]: datetime.datetime(2018, 6, 3, 15, 36, 40, 798523)
```

**Comment by karthikshub** [comment]
Thank you !

**Comment by noviluni** [comment]
This was fixed in the last version `dateparser==1.0.0`: https://github.com/scrapinghub/dateparser/pull/781

Thanks for the feedback :)
